src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/CalculateGenotypePosteriors.java

```diff
@@ -26,30 +26,36 @@
  * Calculate genotype posterior probabilities given family and/or known population genotypes
  *
  * <p>
- * This tool calculates the posterior genotype probability for each sample genotype in a VCF of input variant calls,
- * based on the genotype likelihoods from the samples themselves and, optionally, from input VCFs describing allele
- * frequencies in related populations. The input variants must possess genotype likelihoods generated by
- * HaplotypeCaller, UnifiedGenotyper or another source that provides <b>unbiased</b> genotype likelihoods.</p>
+ * The tool calculates the posterior genotype probability for each sample genotype in a given VCF format callset.
+ * The input variants must present genotype likelihoods generated by HaplotypeCaller, UnifiedGenotyper or other
+ * source that provides unbiased genotype likelihoods.</p>
  *
- * <h4>Statistical notes</h4>
- * <p>The AF field is not used in the calculation as it does not provide a way to estimate the confidence
- * interval or uncertainty around the allele frequency, unlike AN which does provide this necessary information. This
- * uncertainty is modeled by a Dirichlet distribution: that is, the frequency is known up to a Dirichlet distribution
- * with parameters AC1+q,AC2+q,...,(AN-AC1-AC2-...)+q, where "q" is the global frequency prior (typically q << 1). The
- * genotype priors applied then follow a Dirichlet-Multinomial distribution, where 2 alleles per sample are drawn
- * independently. This assumption of independent draws follows from the assumption of Hardy-Weinberg equilibrium (HWE).
- * Thus, HWE is imposed on the likelihoods as a result of CalculateGenotypePosteriors.</p>
+ * <p>
+ * The tool uses priors from three different data sources: (i) one or more supporting germline population callsets
```

Comment:
Can we say something more like "The tool can use priors from three different data sources"?  Most cases won't use all three.

Reply:
What are the annotations, specifically?

The tool will use MLEAC if available or AC if MLEAC is not provided.  AN is also required unless genotypes are provided for all samples.

Reply:
@ldgauthier, the tool document already covers the MLEAC/AC annotations in the bullet point on L51.

Reply:
Changing 'The tool uses priors from' to 'The tool can use priors from'. Will add in the specific annotations exactly as you word it in a section below as it is much clearer than the current bullet point.